Epsilon.Host.Frontend/src/components/CompetenceDocumentDownloadButton.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+    <a @click="downloadCompetenceDocument()">
+      <template v-if="!isDownloading">
+        <span class="icon-wrapper">
+          <font-awesome-icon icon="file-arrow-down" size="lg" alt="download-button-icon" />
+        </span>
+      </template>
+      <template v-else>
+        <RoundLoader class="loading-icon"/>
+      </template>
+      Download document
+    </a>
+  </template>
+  
+  <script lang="ts" setup>
+    import { ref } from 'vue'
+    import { FontAwesomeIcon } from '@fortawesome/vue-fontawesome'
+    import RoundLoader from "@/components/RoundLoader.vue"
+    import axios from 'axios'
+
+    const isDownloading = ref(false);
+  
+    async function downloadCompetenceDocument() {
```

Comment:
Please use the `API` class for requests 
https://github.com/Typiqally/epsilon/blob/develop/Epsilon.Host.Frontend/src/logic/Api.ts